Codespan-reporting 0.13.0 does not compile due to 'use of unstable library feature `inherent_str_constructors`'

Hello!

First of all, many thanks for the nice tool you have built! It's working like a charm. However, with the latest release my build failed due to the following error:

```
error[E0658]: use of unstable library feature `inherent_str_constructors`
  --> /rust/.cargo/registry/xxxxx/codespan-reporting-0.13.0/src/term/mod.rs:48:32
   |
48 |         let buffer_str: &str = str::from_utf8(&buffer).expect("buffer not utf8");
   |                                ^^^^^^^^^^^^^^
   |
   = note: see issue #131114 <https://github.com/rust-lang/rust/issues/131114> for more information
```

Could this be solved with a configuration change on my part?
Thank you very much!

--- a/codespan-reporting/src/term/mod.rs
+++ b/codespan-reporting/src/term/mod.rs
@@ -45,7 +45,7 @@ pub fn emit_to_string<'files, F: Files<'files> + ?Sized>(
             files,
             diagnostic,
         )?;
-        let buffer_str: &str = str::from_utf8(&buffer).expect("buffer not utf8");
+        let buffer_str: &str = core::str::from_utf8(&buffer).expect("buffer not utf8");
         writer.push_str(buffer_str);
         Ok(())
     }

